src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+spring.application.name=player-service
+server.port=9600
+eureka.client.service-url.default-zone=http://localhost:8761/eureka
+# H2
```

Comment:
H2 in Ishigami

Reply:
?

Reply:
Tá certo aqui, a Ishigami não está sendo usada.